File: Board/src/main/java/com/example/board/domain/user/exception/UserNotFoundException.java
package com.example.board.domain.user.exception;

import com.example.board.global.error.exception.CustomException;
import com.example.board.global.error.exception.ErrorCode;

public class UserNotFoundException extends CustomException {

    public static final CustomException EXCEPTION =
            new UserNotFoundException();

    private UserNotFoundException() {
        super(ErrorCode.USER_NOT_FOUND);
    }
}

File: Kopring/Dise_V2/src/main/kotlin/com/example/dise_v2/domain/user/exception/UserNotFoundException.kt
package com.example.dise_v2.domain.user.exception

import com.example.dise_v2.global.error.CustomException
import com.example.dise_v2.global.error.ErrorCode

class UserNotFoundException : CustomException(ErrorCode.USER_NOT_FOUND) {
    companion object {
        @JvmField
        val EXCEPTION = UserNotFoundException()
    }
}